pk/ecc/Makefile

```diff
@@ -2,10 +2,15 @@ CC=gcc
 CFLAGS=-Wall
 LIBS= -lwolfssl
```

Comment:
One last thing sorry it's petty but I like our style.

We've started doing the below, can we keep that trend going?

```
WOLF_PATH=/usr/local
CFLAGS=-I$(WOLF_PATH)/include -Wall
LIBS=-L$(WOLF_PATH)/lib -lwolfssl
```